spec/models/release_spec.rb

```diff
@@ -41,6 +41,33 @@
         run.save!
         expect(run.original_release_version).to eq(expect[:major])
       end
+
+      it "fixed version: sets the original_release_version to train's current version" do
+        train = create(:train, version_seeded_with: ver, freeze_version: true)
+        run = build(:release, original_release_version: nil, train:)
+
+        expect(run.original_release_version).to be_nil
+        run.save!
+        expect(run.original_release_version).to eq(ver)
+      end
+
+      it "fixed version: sets the original_release_version to train's current version with patch_version_bump_only set as true" do
+        train = create(:train, version_seeded_with: ver, freeze_version: true, patch_version_bump_only: true)
```

Comment:
**bug**: this should not be a valid combination. in fact, we should add a validation at the train model level to ensure that only one of `freeze_version` and `patch_version_bump_pnly`  can be true at any given time.